src/routes/articles.js

```diff
@@ -39,10 +39,49 @@ const createArticle = async (req, res, next) => {
 	}
 }
 
+// GET POST BY ID
+const getArticle = async (req, res, next) => {
+	const { id } = req.params
+	try {
+		const article = await getPost({
+			id
+		})
+		if (!article) {
+			return res.status(404).json({
+				success: false,
+				message: 'Article does not exist'
+			})
+		}
+		return res.status(200).json({
+			status: 'success',
+			data: {
+				articleId: article.id,
+				createdOn: article.createdAt,
+				title: article.title,
+				article : article.content,
+				image: article.image,
+				published: article.published,
+				comments: article.comments
+					.filter((comment) => comment)
+					.map((comment) => ({
+						id: comment.id,
+						comment: comment.content,
+						userId: comment.userId
+					}))
+			}
+		})
+	} catch (err) {
+		log.error(err.message)
+		return next(err)
+	}
+}
+
 // ROUTES
 
 router
 	.route('/')
 	.post(isAuthenticated(), createArticle)
-
+router
+	.route('/:id')
```

Comment:
should be authenticated @beblicarl

you can use the `isAuthenticated` like how its used in the posts.js so that it applies to all the routes

Reply:
alright

Reply:
> you can use the isAuthenticated like how its used in the posts.js so that it applies to all the routes

doesn't seem like this was done this way @beblicarl